examples/js/nodes/materials/PhongNode.js

```diff
@@ -131,7 +131,7 @@ THREE.PhongNode.prototype.build = function( builder ) {
 		var normal = this.normal ? this.normal.buildCode( builder, 'v3' ) : undefined;
 		var normalScale = this.normalScale && this.normal ? this.normalScale.buildCode( builder, 'fv1' ) : undefined;
```

Comment:
> Maybe I am missing something, but what I was pointing out is bumpScale is univariate, but normalScale is bivariate.

I had to run some tests. Really works but `normalmap` brings a scale value in .z which also affects the `normal`. 

I did a `vec2` `refraction` with manual scale here: https://github.com/sunag/three.js/commit/157b93e9286d1212e9f5c456ee8297e4a29ada45

but looking better the `textures` I have to put a `vec3` too.